student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, age, name)
```

Comment:
- [ ] Kindly make sure to initialize parent_permission with the given data in the student class, also make sure to add `unknown` value to name.
Example:
`(name = 'unknown', parent_permission: true)`